tools/perf/perf_run.py

```diff
@@ -653,6 +667,8 @@ def run(
             "No Pytorch model (nn.Module) is provided for torchdynamo compilation. Please provide a pytorch model using --model_torch argument"
         )
 
+    torchtrt.runtime.set_cudagraphs_mode(params.get("enable_cuda_graph", False))
```

Comment:
This will only enable cuda graphs for TRT modules. If you want the cuda graphs for whole model (in case of fallback), you need to use with context manager I think.